docs/whirlpool/docs/03-Getting Started/05-Monitor/01-Splash Pool.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_label: Splash Pools
+---
+
+# Fetching a Splash Pool
```

Comment:
I would expect this to come earlier in the guide, maybe after creating a pool?

Reply:
The idea was to place all the 'fetch' functions (pools and positions) under the sidebar category "Monitor", but can also add them under "Create Pool" and "Provide Liquidity" respectively.

Reply:
Yeah think it makes sense to have fetchPool functions together with create pool and fetchPosition function together with provide liquidity!